commands/user_e2e_test.go

```diff
@@ -136,3 +136,36 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestUpdateUserEmailCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("admin and local can change basic user email", func(c client.Client) {
+		newEmail := "basicuser2@fakedomain.com"
+		err := updateUserEmailCmdF(c, &cobra.Command{}, []string{s.th.BasicUser2.Email, newEmail})
+		s.th.BasicUser2.Email = newEmail
+		s.Require().Nil(err)
```

Comment:
As part of the test case checks, we should retrieve the user from the database and check its new email value:

```suggestion
		newUser, err := s.th.App.GetUser(s.th.BasicUser2.Id)
		s.Require().Nil(err)
		s.Require().Equal(newEmail, newUser.Email)
```